Neagu_Anastasia_234_PAO_Project_I.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
[Fix] acest fisier este unul de configurare al intellij-ului si trebuie adaugat in .gitignore. Tot acolo trebuie adaugat si folderul /out cu clase de java compilate local

Reply:
Buna seara! Vad ca in .gitignore am adaugat out/ si *.iml, cum ne-ati aratat in fisierul de lab 0. Imi puteti spune ce altceva mai trebuie sa adaug ca sa ignore acest fisier, va rog?